examples/src/main/kotlin/example/ForwardingRefs.kt

```diff
@@ -0,0 +1,41 @@
+package example
+
+import org.w3c.dom.HTMLInputElement
+import react.*
+import react.dom.html.ReactHTML.input
+
+external interface Focusable {
+    fun focus()
+}
+
+val FancyInput = forwardRef<Props> { _, forwardedRef ->
+    val inputRef = useRef<HTMLInputElement>()
+
+    useImperativeHandle(forwardedRef, inputRef) {
+        object : Focusable {
```

Comment:
`jso`?
Do you really need additional class?

Reply:
yes I do, I want to strictly implement external contract, `jso` is unsafe for it